data/sources/02_rooms-extended.yaml

```
@@ -314,7 +314,7 @@
 "5508.EG.808": { coords: { lat: 48.265906, lon: 11.667831 } }
 "5508.EG.899A": { coords: { lat: 48.265678, lon: 11.667768 } }
 "5508.EG.899B": { coords: { lat: 48.265773, lon: 11.667895 } }
-"5508.01.801": { coords: { lat: 48.265856, lon: 11.667780 } } # Ernst-Schmidt-Hörsaal
+"5508.02.801": { coords: { lat: 48.265856, lon: 11.667780 } } # Ernst-Schmidt-Hörsaal
```

What do you think about this change? This has so far been my main concern for updating TUMonline data, because I am quite sure it is wrong (the lecture hall really is in floor 1, and changing this to 2 would assign it to the second floor, also the map as soon as there is one).

---

If we keep it as is, the new TUMonline integration will fail for this room, so being consistent with TUMonline is more of a concern for us than being correct.

---

The best compromise I can think of at the moment would be to take over the 'wrong' IDs from TUMonline as our own IDs, but patch the room names. E.g. like this:

```suggestion
"5508.02.801":  # ID is probably wrong in TUMonline, but we need to use it for compatibility
  name: "5508.01.801 (Ernst-Schmidt-Hörsaal)"
  coords: { lat: 48.265856, lon: 11.667780 } }
```

This would should make it possible to find the room with both IDs via search, even though it would be irritating. We could further add a warning to the entry page.

---

Fun Fact:
Last time I talked to Philipp von Loewenfeld (The person who maintains the Physics roomfinder) he complained, that the ZA4 gave themselves some “unified guidelines” (temporarily breaking his roomfinder in the process)